content/influxdb/v2.4/reference/release-notes/influx-cli.md

```diff
@@ -8,6 +8,23 @@ menu:
     name: influx CLI 
 ---
 
+## v2.5.0 [2022-10-20]
+
+### Features
+
+- Support username and password in [influx auth create](/influxdb/v2.4/reference/cli/influx/auth/create/).
+
+### Maintenance
+
+- Upgrade to Go 1.19.
+- Fix Go version in `go.mod`.
+
+### Bug fixes
+
+- Don't allow creating an auth with instance resources.
```

Comment:
This is the line that references the commit that fixes the workaround that we added @kelseiv.

Reply:
@jeffreyssmith2nd would it be accurate to say "Fix to allow `influx auth create` to successfully create an authorization without error."?